src/ThemeProvider.js

```diff
@@ -0,0 +1,89 @@
+import React, {PropTypes, Component} from 'react'
+import mitt from 'mitt'
+
+export const CHANNEL = '__glamorous__'
+
+/**
+ * This is a component which will provide a theme to the entire tree
+ * via context and event listener (because pure components block conext updates)
+ * @param {Object} props the props for the component.
+ * @param {Object} context the context for the component.
+ * @return {ReactElement} React.createElement
+ */
+class ThemeProvider extends Component {
+  constructor(props, context) {
+    super(props, context)
```

Comment:
You don't have to pass `props` and `context` to `super` if you're not accessing them on the instance (`this.props`) in the `constructor`